### PR TITLE
Follow typical per-file detail formatting

### PR DESCRIPTION
**Pull request by AbandonedCart**
Not fond of claiming ownership of what is fundamentally just moving code around, but the appearance of the current layout where the version number is appended to the bottom looks awkward. This moves the version number between the title and the developer / distributor, which is the typical layout of individual package information.

### CODE REVIEW

**Comment by lynxnb** [comment]
Appreciate the PR, would you mind posting a screenshot?

**Comment by AbandonedCart** [comment]
Screenshot_20220821-085931_Skyline

It's swapping the last two lines. Really basic.

**Comment by AbandonedCart** [comment]
In defense of the original layout, a lot of "distribution" sites list the version after the publisher because the release date is used to "version" the entire game. Links to the same listing are formatted as the name and version number. Official sites replace the version number with the release date entirely. What is shown to the user is not the complete game as a sum of its parts and release dates are not displayed, so the assumption is that it most resembles a link or file.

You would ideally launch the original file, but with a version number of the latest update (once all that is supported). The original file won't natively have the update number and the update won't be launched directly. It ends up confusing a lot of people explaining that they need to launch 1.0.0 and the update will be applied or you have to pull version numbers from the update files and find the latest to display over the main icon. That is probably why they usually aren't displayed.

**Comment by Uukspppqi** [comment]
Hola skyline

**Comment by AbandonedCart** [comment]
https://github.com/skyline-emu/skyline/commit/5d6eaee301f770245ebfa69307d2d37e5b159c66 explains why it seemed like the author was being checked and version wasn't. I didn't want to go changing code when all I was proposing is restructuring placement.

**Comment by lynxnb** [comment]
I'll be doing a review of this as soon as I find some free time.

**Comment by AbandonedCart** [comment]
Literally moved code from one line to another, so that is probably best taken up with #175.

**Comment by lynxnb** [comment]
Reopened because I do think this PR adds some value even if it's just "moving code around". As maintainers, we can edit code and commits while still keeping you as author, so I can do that if you really have no time.

I was targeting to merge this rather quickly, I'll do a review in a few hours, let me know if you are available or if I should just fix the nits myself.

**Comment by AbandonedCart** [comment]
I wasn't trying to rush you. It was really more or less just that it "looked cleaner" not having a trailing version, but with some semi-rational logic to back it up. It isn't something I wanted anyone wasting more than 5 minutes on and could be circled back to later. Closing a PR doesn't delete it, just takes it out of the pending state.

If it really means that much to you, go ahead and propose your changes. It is because I understand being a maintainer that I favored saving the time.  Not sure if you have any plans to release on Google Play, but wasting time is something they specialize in doing.

File: app/src/main/res/layout/app_item_linear.xml [hunk 1]
```diff
@@ -29,29 +29,30 @@
         android:singleLine="true"
         android:textAppearance="?android:attr/textAppearanceListItem"
         app:layout_constrainedWidth="true"
-        app:layout_constraintBottom_toTopOf="@+id/text_subtitle"
+        app:layout_constraintBottom_toTopOf="@+id/text_version"
```
Comment (lynxnb) [inline app/src/main/res/layout/app_item_linear.xml:32]:
```suggestion
        app:layout_constraintBottom_toTopOf="@id/text_version"
```

File: app/src/main/res/layout/app_item_linear.xml [hunk 2]
```diff
@@ -29,29 +29,30 @@
...
     <TextView
-        android:id="@+id/text_subtitle"
+        android:id="@+id/text_version"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="10dp"
         android:textAppearance="?android:attr/textAppearanceListItemSecondary"
         android:textColor="@android:color/tertiary_text_light"
         app:layout_constrainedWidth="true"
+        app:layout_constraintBottom_toTopOf="@+id/text_subtitle"
```
Comment (lynxnb) [inline app/src/main/res/layout/app_item_linear.xml:47]:
```suggestion
        app:layout_constraintBottom_toTopOf="@id/text_subtitle"
```

File: app/src/main/res/layout/app_item_grid_compact.xml [hunk 3]
```diff
@@ -59,13 +59,13 @@
             android:textAlignment="viewStart"
             android:textAppearance="?android:attr/textAppearanceListItemSecondary"
             android:textColor="@android:color/white"
-            app:layout_constraintBottom_toTopOf="@id/text_version"
+            app:layout_constraintBottom_toTopOf="@+id/text_subtitle"
```
Comment (lynxnb) [inline app/src/main/res/layout/app_item_grid_compact.xml:62]:
```suggestion
            app:layout_constraintBottom_toTopOf="@id/text_subtitle"
```

File: app/src/main/res/layout/app_item_grid_compact.xml [hunk 4]
```diff
@@ -40,13 +40,13 @@
             android:textColor="@android:color/white"
             android:textStyle="bold"
             app:layout_constrainedWidth="true"
-            app:layout_constraintBottom_toTopOf="@id/text_subtitle"
+            app:layout_constraintBottom_toTopOf="@+id/text_version"
```
Comment (lynxnb) [inline app/src/main/res/layout/app_item_grid_compact.xml:43]:
```suggestion
            app:layout_constraintBottom_toTopOf="@id/text_version"
```

**Review by lynxnb — CHANGES_REQUESTED** [review]
Code looks good apart from minor nits, but they are not your fault as they were probably put in previously by Android Studio.

**Comment by AbandonedCart** [comment]
> Could you rename all occurrences of textSubtitle in Kotlin code and game_subtitle in the XML layouts with, respectively, textAuthor and game_author

Assuming you meant textSubtitle and textAuthor to game_subtitle and game_author.

**Comment by AbandonedCart** [comment]
I'm fairly certain it took more time to explain those changes than to have tacked them on, but they're done. I'm not going to move forward with the line changes until someone confirms you are aware it breaks Android standards and adds instability risk. I can inject a commit, but not as a rewrite of the original code for the same reasons.

**Comment by AbandonedCart** [comment]
Had to break those links so every fork or cherry-pick in the world wouldn't throw on a reference to the PR.

**Review by lynxnb — APPROVED** [review]
LGTM, thanks!

**Pull request merged**